js/tji-datasets-explore.js

```diff
@@ -25,9 +25,10 @@ var COLOR_TJI_DEEPPURPLE = '#2D1752';
 // *******************************************************************
 
 
-var TJIGroupByBarChart = function(elt_id, groupBy, data, missing_data_label) {
-  this.elt_id = elt_id;
+var TJIGroupByBarChart = function($container, groupBy, data, missing_data_label) {
+  this.$container = $container;
   this.groupBy = groupBy;
+  this.colors = null;
```

Comment:
Why store colors?

Reply:
you tell me brah, 
![screen shot 2018-07-08 at 3 59 06 pm](https://user-images.githubusercontent.com/3039125/42423888-e3ba4ef2-82c7-11e8-945d-725123e00aff.png)